azurelinuxagent/ga/exthandlers.py

```diff
@@ -131,7 +132,7 @@ def run(self):
             msg = u"Exception retrieving extension handlers: {0}".format(
                 ustr(e))
             logger.warn(msg)
-            add_event(name="WALA", is_success=False, message=msg)
+            add_event(AGENT_NAME, version=CURRENT_VERSION, is_success=False, message=msg)
```

Comment:
I guess this will change it from WALA to WALinuxAgent. Not sure if there's any piece of telemetry/alerting hooked up to this string, just reminding.